pxtcompiler/emitter/service.ts

```diff
@@ -1203,7 +1200,7 @@ namespace ts.pxtc.service {
                     .map(s => completionSymbol(s))
 
                 // in scope locals should be weighter higher
-                inScopePxtSyms.forEach(s => s.weight += 100)
+                inScopePxtSyms.forEach(s => s.weight += 5)
```

Comment:
Not in this PR, but maybe worth going through and setting some constants to make 'visualizing' relative weights easier.